### PR TITLE
Add Elastic-Agent event log documentation

### PR DESCRIPTION
**Pull request by belimawr**
This PR adds documentation for the feature introduced by https://github.com/elastic/elastic-agent/pull/4549

### CODE REVIEW

**Review by kilfoyle — APPROVED** [review]
LGTM! Thanks @belimawr!

**Comment by belimawr** [comment]
I'll wait until https://github.com/elastic/elastic-agent/pull/4549 is merged to merge this PR.

**Review by kilfoyle — DISMISSED** [review]
LGTM! Thanks for adding this @belimawr!

By the way, kind of related to logging, if you have a chance I'd appreciate a review of [this PR](https://github.com/elastic/ingest-docs/pull/489). It's kind of old now. Sorry I lost track of it. :-)

**Comment by belimawr** [comment]
> By the way, kind of related to logging, if you have a chance I'd appreciate a review of [this PR](https://github.com/elastic/ingest-docs/pull/489). It's kind of old now. Sorry I lost track of it. :-)

Approved!

**Comment by jlind23** [comment]
@belimawr can we get this merged? :-)

**Comment by belimawr** [comment]
I'm waiting for the PR with the feature https://github.com/elastic/elastic-agent/pull/4549, to be merged before merging this one.

File: docs/en/ingest-management/elastic-agent/configuration/elastic-agent-standalone-logging.asciidoc [hunk 1]
```diff
@@ -29,6 +29,17 @@ agent.logging.files:
 You can specify the following settings in the Logging section of the
 `elastic-agent.yml` config file.
 
+Some outputs will log raw events on errors like indexing errors in the
```
Comment (cmacknz) [inline docs/en/ingest-management/elastic-agent/configuration/elastic-agent-standalone-logging.asciidoc:32]: You need to document that the content of the event data logs is not shipped to Elasticsearch when agent monitoring is enabled.

Comment (belimawr) [reply docs/en/ingest-management/elastic-agent/configuration/elastic-agent-standalone-logging.asciidoc:32]: I added it on https://github.com/elastic/ingest-docs/pull/1053/commits/a7e3b4ef1d517e32ea21229269d2aad592bf9c82

**Review by kilfoyle — APPROVED** [review]

**Comment by jlind23** [comment]
https://github.com/elastic/elastic-agent/pull/4549 has been merged hence merging this one.

**Pull request merged**